TradeItIosTicketSDK2/TradeItYahooBrokerSelectionViewController.swift

```diff
@@ -91,23 +91,34 @@ class TradeItYahooBrokerSelectionViewController: TradeItYahooViewController, UIT
     }
 
     private func getBroker(atIndexPath indexPath: IndexPath) -> TradeItBroker {
-        if !self.featuredBrokers.isEmpty && indexPath.section == 0 {
+        if self.isFeaturedBrokerSelected(indexPath: indexPath) {
             return self.featuredBrokers[indexPath.row]
         } else {
             return self.brokers[indexPath.row]
         }
     }
 
+    private func isFeaturedBrokerSelected(indexPath: IndexPath) -> Bool {
+        return !self.featuredBrokers.isEmpty && indexPath.section == 0
+    }
+
     // MARK: UITableViewDelegate
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let isFeaturedBrokerSelected = self.isFeaturedBrokerSelected(indexPath: indexPath)
         let selectedBroker = self.getBroker(atIndexPath: indexPath)
+        self.fireLabelTappedEventNotification(
+            view: TradeItNotification.View.selectBroker,
+            title: self.title,
+            labelText: selectedBroker.brokerShortName,
+            label: isFeaturedBrokerSelected ? TradeItNotification.Label.featuredBroker : TradeItNotification.Label.broker
```

Comment:
The use of `label` feels too broad. Are they actually tapping a label? Aren't they cells or buttons? We already have a button event notifier. Maybe these should be called `cellType` and `cellText`? That will make more sense if we end up adding this to all tables.

Thoughts @mitochondrion ?

Reply:
I changed it to label because the featured broker in the welcome screen isn't a row. I can put it back to rowType/rowLabel and didSelectRow/onDidSelectRowNotification, since we remove it from the Welcome screen? (https://github.com/tradingticket/TradeItIosTicketSDK2/pull/323/commits/c176646271a6a8ddebec87e95a41b0b8092d70c3)